Tests/PointFreeTests/SubscribeConfirmationTests.swift

```diff
@@ -355,7 +364,8 @@ class SubscriptionConfirmationTests: TestCase {
           lane: .team,
           billing: nil,
           isOwnerTakingSeat: nil,
-          teammates: nil
+          teammates: nil,
+          referralCode: nil
```

Comment:
is it possible to make a test for when the referral code is present?

Reply:
I'm gonna add tests when the logic is actually in place. Not sure what is possible to test at this point...

Reply:
it's testing the gnarly logic in the `subscriberDataIso` for the route: 

https://github.com/pointfreeco/pointfreeco/pull/570/files#diff-549405fc9f778f602011a17f333a61c5R308

that stuff is still just a hacky workaround to our lack of properly decoding that info from the URL.